src/linagora.esn.calendar/app/components/modals/calendar-secret-address-confirmation/calendar-secret-address-confirmation-modal.pug

```diff
@@ -0,0 +1,10 @@
+.modal.secret-address-confirmation
+  .modal-dialog.modal-sm
+    .modal-content
+      .modal-header
+        h4.modal-title {{ 'Get a new secret address?' | translate }}
+      .modal-body
+        span {{ 'This will make the current secret address invalid(if any).Are you sure want to generate a new secret address? ' | translate:ctrl }}
```

Comment:
```suggestion
        span {{ 'This will make the current secret address invalid (if any). Are you sure want to generate a new secret address? ' | translate }}
```

Reply:
It is also missing translation.